src/coingecko/Coingecko.ts

```diff
@@ -275,6 +270,58 @@ export class Coingecko {
     return this.prices[platform_id][currency];
   }
 
+  protected getCachedAddressPrice(
+    contractAddress: string,
+    currency: string,
+    platform_id: string
+  ): CoinGeckoPrice | undefined {
+    const priceCache = this.getPriceCache(currency, platform_id);
+    const now: number = msToS(Date.now());
+    const tokenPrice: CoinGeckoPrice | undefined = priceCache[contractAddress];
+    if (tokenPrice === undefined || tokenPrice.timestamp + this.maxPriceAge <= now) {
+      if (this.maxPriceAge > 0) {
+        this.logger.debug({
+          at: "Coingecko#getCurrentPriceByContract",
```

Comment:
I think you should change the markers on this and the other (3?) logs from `#getCurrentPriceByContract`  or `#getContractPrices` to `#getCachedAddressPrice` or `#updatePriceCache` (depending on the function they are in).

Reply:
Def, updated here: [3c4ba03](https://github.com/across-protocol/sdk/pull/771/commits/3c4ba03f51d271e34c98042f5d02c2e19d48e141)